react-app/src/components/InputText/index.js

```diff
@@ -95,6 +95,7 @@ export default class InputText extends React.PureComponent {
         <div className={styles.inputContainer}>
           <input
             className={inputStyle}
+            autoCapitalize="off"
```

Comment:
Maybe we can also add
```
autocomplete="off"
autocorrect="off" 
```
to avoid other keyboard default stuff, but this is good, we can add the rest in other PR.

Reply:
I'll add this in this PR in order to avoid the creation of a new one